### PR TITLE
hardware-documentation

### PR DESCRIPTION
**Pull request by cafemoloko**
initial documentation for hardware

### CODE REVIEW

**Review by VanDavv — CHANGES_REQUESTED** [review]
Please remove the `build` dir

File: docs/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+git+git://github.com/sphinx-doc/sphinx.git@8bf84167a30aa05886fcc1ed8895c8c20e939d89
```
Comment (VanDavv) [inline docs/requirements.txt:1]: please update to versions used in this PR - https://github.com/luxonis/depthai-docs-website/pull/172

**Review by VanDavv — CHANGES_REQUESTED** [review]
Other than that, LGTM

**Comment by VanDavv** [comment]
@cafemoloko can you change the index page to something similar to this? (it's a Gen1 landing page - https://docs.luxonis.com/en/gen1_master/)
It would be nice to see the board previews with a link to see their details. 

image

**Comment by cafemoloko** [comment]
> @cafemoloko can you change the index page to something similar to this? (it's a Gen1 landing page - https://docs.luxonis.com/en/gen1_master/)
> It would be nice to see the board previews with a link to see their details.

@VanDavv - working on it. I'm doing something similar to that page, but the boards/parts will be sorted by categories. I'm having a hard time with HTML here :nerd_face:  It looks like Sphinx doesn't read it properly, so I need to come up with a workaround.

**Comment by VanDavv** [comment]
@cafemoloko feel free to HMU offline, did plenty of custom HTML/CSS in Sphinx (and yes, can be tricky sometimes :grin: )

**Comment by cafemoloko** [comment]
> @cafemoloko feel free to HMU offline, did plenty of custom HTML/CSS in Sphinx (and yes, can be tricky sometimes grin )

@VanDavv, I did a new index in pure `rst`. Not the best, but should not crash. We can figure out the custom HTML/CSS that Sphinx digests later.

**Comment by cafemoloko** [comment]
@VanDavv @Luxonis-David - docs new preview [here](https://luxonis-depthai-hardware.readthedocs-hosted.com/en/latest/)

**Comment by Luxonis-Brandon** [comment]
Thanks.  CC: @Luxonis-Brian .

Some quick things I notice:

image

Change this to be: "OAK-D / LUX-D (BW1098OAK)"

Move it so that it is the first entry, the BW1098OBC is second, and keep OAK-1 as third as it is now.

image

**Comment by Luxonis-Brandon** [comment]
On this:

image

Relabel "LDC-1-POE (SJ2096POE)" to OAK-1-POE (SJ2096POE).

Put OAK-D-POE as first on the left, then OAK-1-POE, and POE Board (BW2096POE) as the third.

Relabel "DepthAI for CM3+ (BW1097)" to "DepthAI with Pi CM 3+ (BW1097)" and "DepthAI for Compute module 4 (DM1097)" to "DepthAI with Pi CM 4 (DM1097)"

Move these to be the first 2, with the RPi HAT as the 3rd.

image


And let's update photos of the names on these to be "DepthAI *NEW* FFC (DM1090FFC)" and "DepthAI *OLD* FFC" for "DepthAI USB3 (DM1090FFC)" and "DepthAI USB3 (BW1098FFC)" respectively.

Thoughts?

**Comment by Luxonis-Brandon** [comment]
And @Luxonis-David - would you mind getting @cafemoloko some newer DM1090FFC photos?  @Luxonis-Brian - I can take some light-box photos if you drop one off for me.  :-)

**Comment by Luxonis-Brandon** [comment]
In the meantime here are some DM1090FFC photos: https://photos.app.goo.gl/2HKyk935uZcCw2d36

**Comment by Luxonis-David** [comment]
@Luxonis-Brandon I would love to provide some photos, but I am waiting to receive the boards and don't have any other photos at the moment, so yes maybe best would be if Brian drops one at your place...

**Comment by Luxonis-David** [comment]
@cafemoloko we should also add DM1098OBC_DepthAI_USB3C_WIFI in this section here: 
image

**Comment by Luxonis-David** [comment]
It should be "Altium project files" not "Alium project files" (that is applicable for all boards) 
image

**Review by themarpe — CHANGES_REQUESTED** [review]
My 2 cents:

Not sure if sphinx does any smart image compression, but compress images to `jpeg` if lossless isn't needed and resize to needed resolution depending on context where the image is going to be used, to reduce loading times.

Some outliers:
 - BW1098OAK_USB3C/Images/bw1098oak.png [3.48MB]
 - BW1098OAK_USB3C/Images/bw1098oak.png [4.38MB]
 - ...

**Comment by Luxonis-Brandon** [comment]
Great point.  Yes, we should definitely try to make the images only as large in MB as they absolutely need to be.  For reference on those we should probably shoot for the size to be at or below 0.5MB JPEG.

**Comment by Luxonis-Brian** [comment]
> 
> 
> And @Luxonis-David - would you mind getting @cafemoloko some newer DM1090FFC photos? @Luxonis-Brian - I can take some light-box photos if you drop one off for me. :-)

Yep, I can do that.

**Comment by cafemoloko** [comment]
> My 2 cents:
> 
> Not sure if sphinx does any smart image compression, but compress images to `jpeg` if lossless isn't needed and resize to needed resolution depending on context where the image is going to be used, to reduce loading times.
> 
> Some outliers:
> 
>     * BW1098OAK_USB3C/Images/bw1098oak.png [3.48MB]
> 
>     * BW1098OAK_USB3C/Images/bw1098oak.png [4.38MB]
> 
>     * ...

@themarpe - thanks for noticing, images resized and converted to jpg

**Comment by cafemoloko** [comment]
> In the meantime here are some DM1090FFC photos: https://photos.app.goo.gl/2HKyk935uZcCw2d36

thanks, @Luxonis-Brandon - images are a bit blurry :(

**Review by VanDavv — APPROVED** [review]

**Comment by Luxonis-Brandon** [comment]
Thanks!  Merging and we can do corrections/PRs from there.  :-)

**Pull request merged**